src/components/Text.js

```diff
@@ -20,8 +19,9 @@ const Text = ({
       .text {
         ${weight ? `font-weight: ${weight};` : ''}
         ${fontSizes[size] ? `font-size: ${fontSizes[size]};` : ''}
-        ${lineHeight ? `line-height: ${lineHeight};` : ''}
```

Comment:
lineHeight is still in the propTypes and passed in a lot of places, should this be replaced?